### PR TITLE
Additional step to link multiple Spotify accounts

### PR DESCRIPTION
**Pull request by gde79**
## Proposed change
Add additional required step to link multiple Spotify accounts to newly created apps based on the new Development Mode update.
https://developer.spotify.com/community/news/2021/05/27/improving-the-developer-and-user-experience-for-third-party-apps/

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @gde79, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;home-assistant.io#18524).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: source/_integrations/spotify.markdown [hunk 1]
```diff
@@ -83,7 +83,7 @@ Restart your Home Assistant instance before continuing with the next step.
 This integration supports multiple Spotify accounts at once. You don't need to
 create another Spotify application in the Spotify Developer Portal and no
 modification to the `configuration.yaml` file is needed. Multiple Spotify
-accounts can be linked to a _single_ Spotify application.
+accounts can be linked to a _single_ Spotify application. You will have to add those accounts into the **Users and Access** section of your application in the Spotify Developer Portal.
```
Comment (frenck) [inline source/_integrations/spotify.markdown:86]: That is not needed?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/spotify.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/spotify.markdown:86]: Oh, that is only for new apps :(

Comment (gde79) [reply source/_integrations/spotify.markdown:86]: Correct, else it will give out error 403.

**Review by frenck — APPROVED** [review]
Thanks, @gde79 👍

**Pull request merged**